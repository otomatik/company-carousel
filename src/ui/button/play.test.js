import { Play } from './';
import { shallow } from 'enzyme';
import React from 'react';

describe('Play', () => {
  it('should match snapshot', () => {
    // when
    const button = shallow(<Play />);

    // then
    expect(button).toMatchSnapshot();
  });
});
