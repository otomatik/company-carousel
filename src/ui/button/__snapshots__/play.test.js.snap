// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Play should match snapshot 1`] = `
<styled.button
  alt="Play"
>
  â–¶
</styled.button>
`;
